jdaviz/configs/specviz2d/plugins/spectral_extraction/tests/test_spectral_extraction.py

```diff
@@ -32,9 +32,10 @@ def test_plugin(specviz2d_helper):
     assert len(sp2dv.figure.marks) == 3
 
     pext.keep_active = True
-    assert len(sp2dv.figure.marks) == 12
-    assert pext.marks['trace'].visible is True
-    assert len(pext.marks['trace'].x) > 0
+    # NOTE: UNSKIP THIS NEXT LINE AFTER JDAT-5252 and JDAT-5312!
```

Comment:
```suggestion
    # TODO: UNSKIP THIS NEXT LINE AFTER JDAT-5252 and JDAT-5312!
```
I think this is more standard for comments like this.